src/radarChart.js

```diff
@@ -370,7 +370,7 @@ function displayRADAR(className, options, $element, layout, inputData, self) {
         isNull = true;
       }
     });
-    if(!isNull) {
+    if(!isNull && self.options.noInteraction !== true) {
```

Comment:
(!isNull && self.options.noInteraction !== true) . We can write this as (!isNull && !self.options.noInteraction)